File: app/src/main/java/com/example/amconsofttest/io/realm_models/GeoRealm.java
package com.example.amconsofttest.io.realm_models;

import io.realm.RealmObject;

public class GeoRealm extends RealmObject {

    private String lat;
    private String lng;

    public String getLat() {
        return lat;
    }

    public GeoRealm setLat(String lat) {
        this.lat = lat;
        return this;
    }

    public String getLng() {
        return lng;
    }

    public GeoRealm setLng(String lng) {
        this.lng = lng;
        return this;
    }
}

File: app/src/main/java/com/example/amconsofttest/io/realm_models/GeoRealm.kt
package com.example.amconsofttest.io.realm_models

import io.realm.RealmObject

open class GeoRealm : RealmObject() {

    private var lat: String? = null
    private var lng: String? = null

    fun getLat(): String? {
        return lat
    }

    fun setLat(lat: String): GeoRealm {
        this.lat = lat
        return this
    }

    fun getLng(): String? {
        return lng
    }

    fun setLng(lng: String): GeoRealm {
        this.lng = lng
        return this
    }
}
